app/Models/Article.php

```diff
@@ -105,7 +105,7 @@ public function registerMediaCollections(): void
     {
         $this->addMediaCollection('media')
             ->singleFile()
-            ->acceptsMimeTypes(['image/jpg', 'image/jpeg', 'image/png']);
+            ->acceptsMimeTypes(['image/jpg', 'image/jpeg', 'image/png', 'image/webp', 'image/svg+xml']);
```

Comment:
Remove svg mimes. This is not allowed

Reply:
I removed the svg mimes. It's only the webp right now